migrations/6_deploy_test_contracts.js

```diff
@@ -19,17 +19,17 @@ module.exports = function deploy(deployer, network, accounts) {
       )
       .at(await rootRegistry.getLatestProxyAddr.call('ContractRegistry'));
 
-    const crcMarket = await deployer.deploy(
-      artifacts.require(`./SelectableCrcMarketV0_1_0`)
-    );
-    await crcMarket.initialize(
-      registry.address,
-      [
-        await registry.getLatestProxyAddr.call('CRC'),
-        await registry.getLatestProxyAddr.call('Nori'),
-      ],
-      accounts[0]
-    );
+    // const crcMarket = await deployer.deploy(
```

Comment:
Any reason why this, and the other parts of the testing suite for SelectableCrcMarket, are commented out?

Reply:
Yeah, I talked out the selectable market contracts as they're not relevant to the main app. I left them commented out as a note to myself to make sure to copy the tests over to the atomic-market repo as they're still relevant there. This still needs some clean up before merging, I just opened it to give an idea of how it might work 😁